app/routes.py

```diff
@@ -1,2 +1,50 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+class Planet:
+    def __init__(self, id, name, description, color):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.color = color
+
+    def to_planet_dict(self):
```

Comment:
👀 The live code showed using a name like `to_cat_dict`, so I can see why you used `to_planet_dict` here. But it will be more useful oing forward to all this simply `to_dict` going forward. If we have multiple models in our application, we like to have a method with a standardized name across all of them for converting them into a dictionary.

Consider the `__str__` method that we overrode in Swap Meet. Each item subclass didn't have a differently name version of `__str__` (e.g., `__electronics_str__`). They had a method with the same name so they could all be treated similarly.

So I'd suggest renaming this to `to_dict` and leave the type information to be implied by the instance we're calling this on. Most likely, the planet we're working with would be stored in a variable called `planet`. So with the current name, we end up writing `planet.to_planet_dict()`, which feels a little redundant. If we rename this to `to_dict`, it would become simply `planet.to_dict()`.